### ISSUE TITLE
Remove MOD indicator and other changes

### ISSUE THREAD

**Issue by heavyutils** [scp222thj/MalumMenu#97]
I want the game to look normal when the menu isn't open. Is this possible or can it be made?

**Comment by astra1dev** [comment]
Nope. The Mod badge is required as it says in Innersloth's modding policy

**Comment by heavyutils** [comment]
Was going here to close this issue anyway to say that I just edited the source code to not show it anyway. Along with the new ping indicator.

**Comment by heavyutils** [comment]
> > Nope. The Mod badge is required as it says in Innersloth's modding policy
> 
> 
> 
> Because MalumMenu follows the modding policy?

It doesn't even 😂
Let's just add back the host cheats and allat instead of what we have now.
